scriptworker/github.py

```diff
@@ -119,13 +117,12 @@ def get_tag_hash(self, tag_name):
         url = '/'.join([repo.rstrip('/'), 'branch_commits', revision])
         from scriptworker.task import get_decision_task_id
         cache_key = '{}-{}'.format(get_decision_task_id(context.task), url)
-        async with _branch_commits_cache_lock:
```

Comment:
Status update: I think we need to bring this lock back. See https://bugzilla.mozilla.org/show_bug.cgi?id=1596439 for more details